modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -265,6 +264,9 @@ public class TableManager extends Producer<TableEvent, TableEventParameters> imp
      */
     private final IncrementalVersionedValue<Map<Integer, TableImpl>> tablesByIdVv;
 
+    /** Versioned store for partition storage by table id. */
+    private final IncrementalVersionedValue<Map<Integer, PartitionSet>> partStoragesByIdVv;
```

Comment:
These are not storages, but simply partitions that are currently on the node, the name of the variable and the description are very confusing.